src/bundle/Core/Command/ExpireUserPasswordsCommand.php

```diff
@@ -0,0 +1,383 @@
+                null,
+                InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
+                'Expire passwords of all users based on specific Content Type'
+            )
+            ->addOption(
+                'force',
+                null,
+                InputOption::VALUE_NONE,
+                'Perform setting user passwords as expired'
+            )
+            ->addOption(
+                'batch-size',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Number of users to process at once',
+                self::DEFAULT_BATCH_SIZE
+            )
+            ->addOption(
+                'password-ttl',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'After how many days passwords expire. Set when Content Type needs to be updated.',
+                self::DEFAULT_PASSWORD_TTL
+            )
+            ->setHelp(
+                <<<EOT
+The command <info>%command.name%</info> expires passwords of specific users. 
+Use this tool wisely as next time affect users tries to log in, they will be forced to set a new password.
+Note: This script can potentially run for a very long time, and in Symfony dev environment it will consume memory exponentially with size of dataset.
+
+{$beforeRunningHints}
+EOT
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $batchSize = $input->getOption('batch-size');
+        $userIds = $input->getOption('user-id');
+        $userGroupIds = $input->getOption('user-group-id');
+        $force = $input->getOption('force');
+        $passwordTTL = $input->getOption('password-ttl');
+
+        if (!empty($userIds) && !empty($userGroupIds)) {
+            throw new InvalidArgumentException('You cannot use --user-id and --user-group-id options at once.');
+        }
+
+        $userContentTypeIdentifiers = $input->getOption('user-content-type-identifier');
+
+        $this->summarizeSearchCriteria($output, $userIds, $userGroupIds, $userContentTypeIdentifiers);
+
+        $criteria = $this->getCriteria($userIds, $userGroupIds, $userContentTypeIdentifiers);
+
+        $filter = new Filter(new LogicalAnd($criteria));
+        $filter->withLimit(0);
+
+        $contentList = $this->repository->sudo(static function (Repository $repository) use ($filter): ContentList {
+            return $repository->getContentService()->find($filter);
+        });
+        $count = $contentList->getTotalCount();
+
+        if ($count === 0) {
+            $output->writeln('<info>There are no users matching given criteria</info>');
+
+            return Command::SUCCESS;
+        }
+
+        $output->writeln(sprintf(
+            '<info>Found %d User(s) matching given criteria</info>',
+            $count
+        ));
+
+        $displayProgressBar = !($output->isVerbose() || $output->isVeryVerbose() || $output->isDebug());
```

Comment:
By default on normal output progress bar is displayed. If you want it not to be displayed, you typically execute your command with `-q` switch. This is controlled by:

`\Symfony\Component\Console\Output\Output::write:`
```php
        if ($verbosity > $this->getVerbosity()) {
            return;
        }
```

and TBH I don't see a reason we should "mingle" with Symfony's default behavior

Reply:
It was inspired by one of the other commands. But I agree with you and fixed it.